### ISSUE TITLE
Long Press/Hold Popup Menu

### ISSUE THREAD

**Issue by Raul1198** [fork-maintainers/iceraven-browser#124]
### What is the user problem or growth opportunity you want to see solved?
the long press menu should follow the tapped location the device instead of always on center . here i use Via Browser as a reference( also Telegram can used) for long press option without scrolling the page

1. About  -

![about m](https://user-images.githubusercontent.com/29801315/93496138-7f6dc900-f92c-11ea-9fb9-49d4266074b1.jpg)
 VS 
![about v](https://user-images.githubusercontent.com/29801315/93496188-8b598b00-f92c-11ea-8c5b-b96187be2f7f.jpg)

2. Mozilla Link -

![moz m](https://user-images.githubusercontent.com/29801315/93496243-9ad8d400-f92c-11ea-9a42-0e434e364039.jpg)
VS
![moz v](https://user-images.githubusercontent.com/29801315/93496269-a1674b80-f92c-11ea-9216-79b4ccb15305.jpg)

3. Release -

![release m](https://user-images.githubusercontent.com/29801315/93496363-bba12980-f92c-11ea-8ee8-150a1edd8a97.jpg)
 VS
![release v](https://user-images.githubusercontent.com/29801315/93496380-bfcd4700-f92c-11ea-8aeb-ef013ee817eb.jpg)

### How do you know that this problem exists today? Why is this important?

after using Telegram's tap on a post/message the tap's menu follow the tap location

### Who will benefit from it?
users with big display

**Comment by interfect** [comment]
Yeah, the current behavior (context menu always centered) can't make any sense for e.g. tablet use.
